engines/engine.h

```diff
@@ -587,9 +587,16 @@ class Engine {
 	inline Common::SaveFileManager *getSaveFileManager() { return _saveFileMan; }
 
 public:
-	/** On some systems, check whether the game appears to be run from CD. */
-	void checkCD();
-
+	/** Check if extraced CD Audio files are found.
+	*/
```

Comment:
Nitpicking: Multi-line comments should be like this:



    /**
      * Check if extracted CD Audio files are found.
      */

Reply:
Also, it's "extracted"

Reply:
Fixed in commit: https://github.com/scummvm/scummvm/pull/3018/commits/f76acd499e81a06ec603e3beed8e97114d30230e